### PR TITLE
Make sure all Wikibase codebases run PHPCS during `composer test`

### PR DESCRIPTION
**Pull request by bereket-WMDE**
Bug: [T188395](https://phabricator.wikimedia.org/T188395)

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Boo. https://phabricator.wikimedia.org/T188395#4651556

File: composer.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
 	"scripts": {
 		"test": [
 			"composer validate --no-interaction",
+			"composer cs",
```
Comment (tzhelyazkova) [inline composer.json:57]: this can alternatively be `@cs`

**Review by tzhelyazkova — CHANGES_REQUESTED** [review]
the `@cs` command should be removed from the `ci` script to avoid running it twice (since it's part of `test` now)

File: composer.json (hunk 1 above)
Comment (bereket-WMDE) [reply composer.json]: thanks. I updated the changes

**Pull request merged**